internal/config/config.go

```diff
@@ -37,7 +37,7 @@ type Config struct {
 // New creates a new config with sane default values.
 func New() *Config {
 	return &Config{
-		AutoImport:    true,
+		AutoImport:    false,
```

Comment:
This might warrant a warning in the RELEASE_NOTES, no?

Reply:
Done